### PR TITLE
Allow Symfony 2.6 components + remove composer as dev dependency

### PR DESCRIPTION
**Pull request by AydinHassan**
**should** fix https://github.com/php-school/learn-you-php/issues/43

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,21 +17,20 @@
         "php-di/php-di": "^5.0",
         "php-di/invoker": "^1.0",
         "container-interop/container-interop": "^1.0",
-        "symfony/process": "^2.7|^3.0",
```
Comment (renatomefidf) [inline composer.json]: in this case you can use just ^2.6|^3.0. Because it will up to 2.9 and 3.9

**Comment by renatomefidf** [comment]
Even though the tests failed, seems like sf 2.6 is ok :)

File: composer.json (hunk 1 above)
Comment (AydinHassan) [inline composer.json]: Cool thanks, will update!

**Comment by AydinHassan** [comment]
Yeah I thought we were not using composer as a dev dependency. Seems I forget code I wrote last week haha.

**Comment by renatomefidf** [comment]
@AydinHassan also can you try with sf 2.3?

I think 2.3 was the one introducing some new stuff, so maybe it worth to try!

**Comment by AydinHassan** [comment]
:+1:

**Comment by renatomefidf** [comment]
Seems to be working, the failing test has an weird composer error, maybe retry is enough!

@AydinHassan How can I help better in the project? I'm interested on it!

**Comment by AydinHassan** [comment]
@renatomefidf Yeah that was a weird error, something to do with Composer cache? I've restarted it. Help would be great, I think it would be awesome to build some more workshops, but that kind of relies on me writing documentation for this package, which I will be doing over the next week. 

Other than that, hanging around and just helping with issues and stuff, just whatever you can do is greatly appreciated!

**Comment by AydinHassan** [comment]
Hmm I've restarted 3 times and still the same, I think it's cache related. I'll jump back on this after work :+1:

**Comment by renatomefidf** [comment]
@AydinHassan thanks! I'll stay around!
Do you have a chat or something where you discuss the new ideas?

Its weird indeed and seems to be cache :/

**Comment by AydinHassan** [comment]
yep: http://www.phpschool.io/ Look for the Slack link :)

**Comment by renatomefidf** [comment]
@AydinHassan https://docs.travis-ci.com/user/caching/#Clearing-Caches

**Comment by AydinHassan** [comment]
Great, that sorted it

**Comment by AydinHassan** [comment]
Will tag later tonight, gotta run out for a few hours.

**Comment by AydinHassan** [comment]
Just tagged 0.3.6 :tada:

**Pull request merged**